src/components/table/sort.ts

```diff
@@ -40,9 +42,9 @@ function isEmptyValue(value?: string): boolean {
   }
 }
 
-function descendingNumComparator<T>(a: T, b: T, orderBy: keyof T): number | null {
-  const aNumValue = Number((a[orderBy] ?? '0.0') as string);
-  const bNumValue = Number((b[orderBy] ?? '0.0') as string);
+function descendingNumComparator(aValue: any, bValue: any): number | null {
```

Comment:
```suggestion
function descendingNumComparator(aValue: unknown, bValue: uknown): number | null {
```

`unknown` is better to use than `any` if you are unsure what something is. `any` allows anything, whereas `unknown` forces type checking where necessary.